TypeCobol/Parser.cs

```diff
@@ -26,8 +26,8 @@ public class Parser
 		/// <summary>Optional custom symbol table to use for name and type resolution.</summary>
 		public SymbolTable CustomSymbols = null;
 
-		public string[] Extensions = { ".cbl", ".cpy" };
-		public string[] CopyExtensions = { ".cpy" };
+		public string[] Extensions = { ".cbl", ".cpy", ".copy" };
+		public string[] CopyExtensions = { ".cpy", ".copy" };
```

Comment:
Workspace.cs should also need to be modified.

I think we should have only one constant for copy extensions. Something like `DEFAULT_COPY_EXTENSIONS`
Maybe this constant could be in Helpers or in CompilationProject.

Reply:
Done